torch/csrc/autograd/variable.h

```diff
@@ -275,6 +278,21 @@ inline bool& Variable::is_volatile() {
   return get()->is_volatile;
 }
 
+inline Variable Variable::detach() const {
```

Comment:
Can you move these definitions to `variable.cpp`? (And remove the inline flag). I'm worried that as variable.h gets bigger compilation time will slow down since it's used everywhere.